modules/ROOT/pages/organization.adoc

```diff
@@ -139,11 +160,15 @@ If you are not an organization administrator, you can view only the business gro
 
 === Creating Roles and Handling Membership to Business Groups
 
-To obtain the membership for a business group, you must be granted a role within that business group. Members that are added to a business group can view and access the business group.
+Roles can be applied at the master organization level as well as at the business group level. However, each controls different resources.
+
+To obtain the membership for a business group, you must be granted a role within that business group. Once you are visible in a business group, you can be granted individual permissions in that business group. Because users are always visible in the master organization, they can always be assigned individual permissions in the master organization.
```

Comment:
i would not use "membership in a business group": Users always belong to the master organization, regardless of which business group they were invited to. Therefore, i'd use "visibility in a business group" instead of "membership"

Reply:
Hi Gerald,
For now I am going to leave it as is. Changes to terminology require a broader discussion with more stake holders. But maybe I can rework it a little so it's more clear that users always belong to the master organization.